web/src/views/FileBrowserView/FileBrowser.vue

```diff
@@ -388,7 +392,7 @@ function getExternalServices(path: AssetPath) {
     .filter((service) => servicePredicate(service, path))
     .map((service) => ({
       name: service.name,
-      url: `${service.endpoint}${trimEnd((path.asset as AssetFile).url, '/')}`,
+      url: `${service.endpoint}${trimEnd((path.asset as AssetFile).url, '/')}&dandi-asset=${assetMetadataURI((path.asset as AssetFile).asset_id)}`,
```

Comment:
why not just provide as a separate kwarg to be used in definition of the url to jump to in the service? i.e.

```suggestion
      url: `${service.endpoint}${trimEnd((path.asset as AssetFile).url, '/')}`,
      asset_id: `${assetMetadataURI((path.asset as AssetFile).asset_id)}`,
```
?

Reply:
AH -- I just now saw that we didn't follow the original design idea here and had only prefix for the url:

```
  {
    name: 'Neurosift',
    regex: /\.nwb$/,
    maxsize: Infinity,
    endpoint: 'https://flatironinstitute.github.io/neurosift?p=/nwb&url=',
  },
```

my bad -- we ideally would need to RF it to provide all those "format" kwargs, so here it would become 

```
  {
    name: 'Neurosift',
    regex: /\.nwb$/,
    maxsize: Infinity,
    endpoint: 'https://flatironinstitute.github.io/neurosift?p=/nwb&url={url}&dandi-asset={asset_id}',
  },
```

or some other way java script allows for such embeddings...

Reply:
according to chatgpt JS/vue has similar to python f-strings so we can easily do that.

```
const struct = { field1: "value" };
const v = `something ${struct.field1}`;
```